readme.md

```diff
@@ -1,58 +1,119 @@
+Binary packages can be downloads from
+[Releases](https://github.com/pawn-lang/compiler/releases).
 
-You can also get the latest development binaries for Windows on [AppVeyor](https://ci.appveyor.com/project/pawn-lang/compiler/branch/master/artifacts). This archive is built automatically on every Git commit and can be pretty unstable, so use at your own risk.
+You can also get the latest development binaries for Windows on
+[AppVeyor](https://ci.appveyor.com/project/pawn-lang/compiler/branch/master/artifacts).
+This archive is built automatically on every Git commit and can be pretty
+unstable, so use at your own risk.
 
-## Installation on Windows
+### Windows
 
-Download the ZIP archive and extract `pawnc.dll`, `pawncc.exe`, `pawndisasmsm.exe` to your pawno directory or another directory of your choice if you're using a different editor.
+If you just use an editor configured to run `pawncc` such as Pawno, Sublime Text
+or VS Code you can simply delete your existing `pawncc.exe` and replace it with
+the new one.
 
-## Installation on openSUSE/SLES
+Download the ZIP archive and extract `pawnc.dll`, `pawncc.exe`,
+`pawndisasmsm.exe` to your original `pawncc` directory. If you use Pawno, this
+will be in the `pawno/` folder that's inside your server directory.
 
-There is an installation package available for openSUSE/SLES users so that you can easily install the compiler on your distribution. Please follow these steps:
+### openSUSE/SLES
 
-1. Go to https://build.opensuse.org/package/show/home:mschnitzer/pawncc
-2. On the right side, select your distribution (only if it's not disabled!)
-3. Click "Go to download repository"
-4. Copy the link and enter in sudo mode in your shell: `zypper ar $COPIED_LINK home:mschnitzer`
-5. Again as root, type: `zypper ref`
-6. Install the package with `zypper in pawncc`
-7. Run `pawncc` in your shell to test if it's working
+There is an installation package available for openSUSE/SLES users so that you
+can easily install the compiler on your distribution. Please follow these steps:
+
+1.  Go to https://build.opensuse.org/package/show/home:mschnitzer/pawncc
+2.  On the right side, select your distribution (only if it's not disabled!)
+3.  Click "Go to download repository"
+4.  Copy the link and enter in sudo mode in your shell:
+    `zypper ar $COPIED_LINK home:mschnitzer`
+5.  Again as root, type: `zypper ref`
+6.  Install the package with `zypper in pawncc`
+7.  Run `pawncc` in your shell to test if it's working
+
+### With sampctl
+
+If you are a sampctl user, simply set the `version` field in `build`/`builds`:
+
+```json
+{
+  "builds": [
+    {
+      "name": "production",
+      "version": "3.10.8"
+    }
+  ]
+}
+```
 
 ## Building from source code
 
-In general you will need [CMake](https://cmake.org/) and a C compiller to build Pawn from source code.
+In general you will need [CMake](https://cmake.org/) and a C compiller to build
+Pawn from source code.
 
 ### Building on Windows
 
```

Comment:
Visual Studio 2017 and later has CMake integrated into the IDE. The folder containing CMakeLists.txt has to be opened in Visual Studio. The IDE will automatically generate the cache and allow you to build in one click.

Reply:
Yes I will update this with that info. It's far easier than doing the generation beforehand!